charters/UNITY/charter.md

```diff
@@ -0,0 +1,54 @@
+# Unity Benchmarking Service
+
+## Description
+
+The Unity benchmarking service automates the process of running and benchmarking pipelines.
+
+## Objectives
+
+The Unity benchmarking service aims to automate benchmarking for HCA pipelines, enforce standards for consistent reporting for historic comparison and enable the public to participate in suggesting improvements to pipelines by empowering them to implement improvements and test results using the same infrastructure as the HCA data processing pipelines team. The Unity benchmarking service will be used by those making pipelines for the HCA data processing service and will be public, so anyone can benchmark their implemented ideas against HCA pipelines.
+
+## In-scope
+
+* Benchmarking for HCA pipelines will be automated and included in Unity
+* Benchmarking data will be available through Unity but not controllable by the users
+* Benchmarking data can change without notice but will be consistent in format and schema with previous data or notice will be given to users
+* Leaderboards showing how pipelines differ will be created as different pipelines for the same assay are collected
+* Documentation on how to use Unity and the assumptions pipelines must meet to be ran using Unity
```

Comment:
The _assumptions pipelines must meet to be ran_ was hard for me to parse. 

 I especially don't understand the use of _assumptions_ in this context. Are there not requirements? 

Perhaps - _Documentation for the requirements of pipelines to be benchmarked by the Unity Benchmarking Service._

Reply:
love it, adding directly.